src/models/vcard.js

```diff
@@ -41,6 +41,7 @@ export class VCard extends DavObject {
 		super(...args);
 
 		super._exposeProperty('data', NS.IETF_CARDDAV, 'address-data', true);
+		super._exposeProperty('hasphoto', NS.NEXTCLOUD, 'has-photo', false);
```

Comment:
@skjnldsv I feel like this should be camelCase, but I will leave that up to you.

Reply:
I assumed not since contenttype is not :man_shrugging: 
![Capture d’écran_2019-03-27_12-20-58](https://user-images.githubusercontent.com/14975046/55072227-d3c32000-508a-11e9-9b65-69e3e0d9a7e4.png)
https://github.com/nextcloud/cdav-library/blob/b7733343f2bd8a7e4284ba7fce499d2f8fc06430/src/models/davObject.js#L59